pkg/apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -83,6 +86,53 @@ type AuthEnvironment struct {
 	ContainerName string `json:"containerName"`
 }
 
+// HashiCorpVault is used to authenticate using Hashicorp Vault
+// +k8s:openapi-gen=true
+type HashiCorpVault struct {
+	Address        string              `json:"address"`
+	Authentication VaultAuthentication `json:"authentication"`
+
+	// +listType
+	Secrets []VaultSecret `json:"secrets"`
+
+	// +optional
+	Credential Credential `json:"credetial"`
```

Comment:
It looks like there is a misspelling here that carries through in a couple other spots

Reply:
thanks a lot, updated :)